src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+import duke.storage.Storage;
+import duke.ui.Display;
+import duke.ui.Parser;
+
+public class Duke {
```

Comment:
I feel that it would be better if header comments for public classes/methods were included. I noticed the same issue in several other places too.